ray-curator/ray_curator/examples/video/video_split_clip_example.py

```diff
@@ -151,7 +156,8 @@ def main(args: argparse.Namespace) -> None:
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     # General arguments
-    parser.add_argument("--video-folder", type=str, default="/home/aot/Videos")
+    parser.add_argument("--video-dir", type=str, required=True, help="Path to input video directory")
+    parser.add_argument("--model-dir", type=str, required=True, help="Path to model directory")
```

Comment:
Why is this required? We would download if they don't provide this? Or do we have similar interface as cosmos curate where we first download in a seperate step?

Reply:
For now we required to have the model predownloaded in order to make all the directory mapping works. We can refactor this later for `setup_on_node `